taxonomy-editor-frontend/src/pages/startproject/index.tsx

```diff
@@ -22,6 +22,19 @@ import { createBaseURL, toSnakeCase } from "@/utils";
 
 const branchNameRegEx = /[^a-z0-9_]+/;
 
+function formatDate(date) {
+  const map = {
+    mm: date.getMonth() + 1,
+    dd: date.getDate(),
```

Comment:
nit: I would make it so that these variables are always 2 characters long, even if the month or day is < 10 (I mean display 03 instead of 3 for March for instance, so that the format and number of characters is the same for all branches and so more readable)